Formula/meme.rb

```diff
@@ -24,8 +22,8 @@ class Meme < Formula
   end
 
   def install
-    ENV.deparallelize
-    system "./configure", "--disable-debug", "--disable-dependency-tracking", "--prefix=#{libexec}"
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{libexec}", "--with-url=http://meme-suite.org/", "--enable-build-libxml2", "--enable-build-libxslt"
+    system "make", "-j#{ENV.make_jobs}"
```

Comment:
`ENV.make_jobs` is added automatically so shouldn't be needed.

Reply:
```suggestion
```
Thank you for the suggestion. I didn't know that.